routes/user.js

```diff
@@ -0,0 +1,65 @@
+const express = require('express'),
+    passport = require('passport'),
+    ensureLoggedIn = require('connect-ensure-login').ensureLoggedIn(),
+    router = express.Router();
+
+/* GET user profile. */
+router.get('/', ensureLoggedIn, (req, res, next) => {
+  res.render('user', {
+    user: req.user
+  });
+});
+
+router.get('/profile', ensureLoggedIn, (req, res, next) => {
+  res.render('profile'); 
+});
+
+router.post('/profile', ensureLoggedIn, (req, res, next) => {
+    const {firstName, surname, streetAddress, city, state, zip} = req.body;
+    let errors = [];
+
+    if(!firstName && !firstName.trim()) {
```

Comment:
tutaj powinno być chyba || a nie && bo error powinien być jeśli nie ma firstName *lub* firstName to same biale znaki